JustSaying/AwsTools/JustSayingConstants.cs

```diff
@@ -2,70 +2,70 @@ namespace JustSaying.AwsTools
 {
     public static class JustSayingConstants
     {
-        public const string ATTRIBUTE_REDRIVE_POLICY = "RedrivePolicy";
-        public const string ATTRIBUTE_ARN = "QueueArn";
-        public const string ATTRIBUTE_RETENTION_PERIOD = "MessageRetentionPeriod";
-        public const string ATTRIBUTE_VISIBILITY_TIMEOUT = "VisibilityTimeout";
-        public const string ATTRIBUTE_DELIVERY_DELAY = "DelaySeconds";
-        public const string ATTRIBUTE_POLICY = "Policy";
-        public const string ATTRIBUTE_ENCRYPTION_KEY_ID = "KmsMasterKeyId";
-        public const string ATTRIBUTE_ENCRYPTION_KEY_REUSE_PERIOD_SECOND_ID = "KmsDataKeyReusePeriodSeconds";
+        public const string AttributeRedrivePolicy = "RedrivePolicy";
+        public const string AttributeArn = "QueueArn";
+        public const string AttributeRetentionPeriod = "MessageRetentionPeriod";
+        public const string AttributeVisibilityTimeout = "VisibilityTimeout";
+        public const string AttributeDeliveryDelay = "DelaySeconds";
+        public const string AttributePolicy = "Policy";
+        public const string AttributeEncryptionKeyId = "KmsMasterKeyId";
+        public const string AttributeEncryptionKeyReusePeriodSecondId = "KmsDataKeyReusePeriodSeconds";
 
         /// <summary>
         /// Default visibility timeout for message in seconds
         /// </summary>
-        public static int DEFAULT_VISIBILITY_TIMEOUT = 30;
+        public static int DefaultVisibilityTimeout = 30;
         
         /// <summary>
         /// Number of times a handler will retry a message until a message 
         /// is sent to error queue
         /// </summary>
-        public static int DEFAULT_HANDLER_RETRY_COUNT = 5;
+        public static int DefaultHandlerRetryCount = 5;
 
         /// <summary>
         /// Number of times publisher will retry to publish a message if destination is down.
         /// </summary>
-        public static int DEFAULT_PUBLISHER_RETRY_COUNT = 3;
+        public static int DefaultPublisherRetryCount = 3;
 
         /// <summary>
         /// Every time a publisher is not able to deliver a message, it will 
         /// wait {interval}*{attemptCount} miliseconds before retrying,
         /// </summary>
-        public static int DEFAULT_PUBLISHER_RETRY_INTERVAL = 100;//100 milliseconds
+        public static int DefaultPublisherRetryInterval = 100;//100 milliseconds
```

Comment:
Just an observation, it's worrying that in the same constants we use different units of time for constants. Some are seconds, some are milliseconds. That's almost asking for someone to make a mistake.

Reply:
Seperate from naming, but we can store them all as what? `Milliseconds`? `Timespan`?

Reply:
Personally I prefer `TimeSpan` as they can support more-or-less arbitrary precision, and are easy to unambiguously define, such as with `TimeSpan.FromMinutes(2)`.

Reply:
Agreed, though if we e.g. change `int JustSayingConstants.DefaultVisibilityTimeout` to  `TimeSpan JustSayingConstants.DefaultVisibilityTimeout` then we will also want to change `SqsReadConfiguration.VisibilityTimeoutSeconds` as that's what it's a default for.

Reply:
Personally I'm not concerned _too_ much with the ripple effects if we're redesigning the API to make it "nicer" and those are the consequences.